### DISCUSSION TITLE
Transfer project between organizations?

### DISCUSSION THREAD

**Discussion by ilangleben19** [supabase/supabase#2471] [Questions]
Hi,

I have two organizations under my account, and a project currently attached to organization A. I'd like to transfer it to organization B so that users on the organization B team can access it.

Is there any way to do this? Any help is appreciated!

**Comment by silentworks** [comment]
I don't think there is anything in the UI to do this at the moment, but maybe you can drop the Supabase team an email and see if they can do it internally.
